clang/lib/AST/Type.cpp

```diff
@@ -2863,6 +2863,15 @@ bool QualType::isWebAssemblyFuncrefType() const {
          getAddressSpace() == LangAS::wasm_funcref;
 }
 
+bool QualType::hasWrapsAttr() const {
+  return !isNull() && getTypePtr()->hasAttr(attr::Wraps) &&
```

Comment:
we probably want to do 'more work' to pull out the type you want/canonicalize the type here.